desc/objectives/utils.py

```diff
@@ -83,12 +87,89 @@ def get_fixed_boundary_constraints(
     return constraints
 
 
+def get_fixed_axis_constraints(profiles=True, iota=True):
+    """Get the constraints necessary for a fixed-axis equilibrium problem.
+
+    Parameters
+    ----------
+    profiles : bool
+        Whether to also return constraints to fix input profiles.
+    iota : bool
+        Whether to add FixIota or FixCurrent as a constraint.
```

Comment:
Would it be cleaner to have the inputs be `iota=False, current=False` instead of `profiles` and `iota`? That feels more intuitive to me, but the advantage of the existing code is that it prevents having both profiles constrained.

Reply:
I was just sticking in with the convention we already have, as Rory said